client/utils.js

```diff
@@ -17,3 +17,27 @@ export function walkModules(modules, cb) {
 export function elementIsOutside(elem, container) {
   return !(elem === container || container.contains(elem));
 }
+
+export const localStorage = {
+
+  getItem(k) {
+    try {
+      return window.localStorage.getItem(k);
```

Comment:
Let's create a constant that will contain a prefix e.g. `wba` and prepend it to all keys. 
Let's also `JSON.stringify/parse` values so we can store any primitive value, not just string.

This is how I see this function:
```js
getItem(key) {
  try {
    return JSON.parse(window.localStorage.getItem(`${KEY_PREFIX}.${key}`));
  catch (err) {
    return null;
  }
}
```